medusa/trakt_checker.py

```diff
@@ -382,12 +381,11 @@ def add_episode_watchlist(self):
         if app.TRAKT_SYNC_WATCHLIST and app.USE_TRAKT:
 
             main_db_con = db.DBConnection()
-            status = Quality.SNATCHED + Quality.SNATCHED_BEST + Quality.SNATCHED_PROPER + [WANTED]
-            selection_status = [b'?' for _ in status]
+            status = [SNATCHED, SNATCHED_BEST, SNATCHED_PROPER, WANTED]
             sql_selection = b'SELECT s.indexer, s.startyear, e.showid, s.show_name, e.season, e.episode ' \
                             b'FROM tv_episodes AS e, tv_shows AS s ' \
                             b'WHERE e.indexer = s.indexer AND s.indexer_id = e.showid AND s.paused = 0 ' \
-                            b'AND e.status in ({0})'.format(b','.join(selection_status))
+                            b'AND e.status in ({0})'.format(b','.join([b'?']*len(status)))
```

Comment:
Same as above.

Reply:
That's what it says right? Do you mean the binary string conversion? That's just there to make sure the end string is also binary.

Reply:
No, sorry. And I didn't even notice I dropped the `b` prefix.
@medariox it wasn't intentional.
I meant the spaces around the asterisk.